rbi/pdf-reader.rbi

```diff
@@ -1110,7 +1110,10 @@ module PDF
           y: Numeric,
         ).void
       end
-      def initialize(x, y); end
+      def initialize(x, y)
+        @x = T.let(0, Numeric)
```

Comment:
You don't need the actual value of the variable in RBI files, you should be able to just do:
```ruby
@x = T.let(T.unsafe(nil), Numeric)
```
instead. Here `T.unsafe(nil)` creates a dummy value that is `T.untyped` and `T.let` declares the type of it.

This way, your internal implementation details will not leak into RBI files, you can just declare the types instead.

Reply:
cheers, updated in #416